src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx

```diff
@@ -39,9 +40,9 @@ export const ClaimNoteItem: React.FC<{
     insertedBy?.__typename === 'MemberSystemUser'
   const noteType: 'text' | 'markdown' = isSystemNote ? 'text' : 'markdown'
   let noteText = <NoteText>{text}</NoteText>
-  if (noteType == 'markdown') {
+  if (noteType === 'markdown') {
     noteText = (
-      <div>
+      <div className={css.richText}>
         <Markdown>{text}</Markdown>
       </div>
```

Comment:
Can we pass the `className` to `Markdown` directly?
```suggestion
    <Markdown className={css.richText}>{text}</Markdown>
```

Reply:
Markdown does not produce root element, so we need to insert our own